pkgs/by-name/pa/paperjam/package.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenv,
+  fetchgit,
+  qpdf,
+  libiconv,
+  libpaper,
+  asciidoc,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "paperjam";
+  version = "1.2-unstable-2023-05-13";
+
+  src = fetchgit {
+    url = "git://git.ucw.cz/paperjam.git";
+    # this is v1.2 + build fix
+    rev = "65444824e308470f49b5e4f98fd1763b7da3bd24";
+    hash = "sha256-AMoSiHDKbnVtmNR07sniXruVKH7p/Pl5aey4SF8aMWA=";
+  };
+
+  buildInputs = [
+    qpdf
+    libpaper
+    asciidoc
+  ] ++ lib.optional stdenv.hostPlatform.isDarwin libiconv;
```

Comment:
The build still fails for some reason:
```
Undefined symbols for architecture x86_64:
  "_iconv", referenced from:
      unicode_string(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>) in pdf-tools.o
  "_iconv_close", referenced from:
      unicode_string(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>) in pdf-tools.o
  "_iconv_open", referenced from:
      unicode_string(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>) in pdf-tools.o
ld: symbol(s) not found for architecture x86_64
```

Reply:
Oh, yes of course. Providing the dependency is not enough, one also has to link to it.

Also, I just noticed that in the meantime version 1.2.1 was released; so I updated to that.